### DISCUSSION TITLE
First time Mesa User: Can't use self.model.grid.get_neighbors over an arbitrary value.

### DISCUSSION THREAD

**Discussion by Jomomo05** [mesa/mesa#1532] [Q&A] [answered]
As shown on: https://mesa.readthedocs.io/en/latest/apis/space.html#mesa.space.MultiGrid.get_neighbors 

I was trying to use

`vecinos = self.model.grid.get_neighbors(
        (3,3)
        moore=True,
        include_center=False)
        `
        To get the neighbors over (3,3).
        
        But, when i try to run this code, i got multiple Tuple errors.
        
        I added "from typing import Tuple" and tried to use typing.Tuple[3,3] but it didn't work too.
        
        I am stuck and out of ideas.
        
        Any support would be appreciated.

**Comment by rht** [comment]
> `vecinos = self.model.grid.get_neighbors( (3,3) moore=True, include_center=False)`

It looks like there is no comma in between `(3,3)` and `moore=True`? Is the typo present in the code?

**Reply by Jomomo05** [reply]
I am sorry, this typo isn't present on my code.

My code right now is:

`vecinos = self.model.grid.get_neighbors(
        (3,3),
        moore=True,
        include_center=False)`

**Reply by rht** [reply]
May I know the exact error message?

**Reply by Jomomo05** [reply]
Of course.

The full error message is: 

![image](https://user-images.githubusercontent.com/69222925/202583345-c2d7be7c-d144-40e0-894f-7196f47c65b9.png)


Also, i attach my ipynb.
[Tituloalv (1) (1) (3).zip](https://github.com/projectmesa/mesa/files/10036531/Tituloalv.1.1.3.zip)

**Comment by rht** [comment, marked as answer]
The error happened because `self.nueva_posicion` is `None` instead of a tuple. If you handle this case in `advance`, there should be no problem. Make sure to also check that `if self.pos != self.nueva_posicion` also makes sense if it is `None`.
